Signal/src/ViewControllers/ConversationView/Cells/OWSAudioMessageView.m

```diff
@@ -140,7 +140,7 @@ - (CGFloat)hSpacing
 
 + (CGFloat)vMargin
 {
-    return 0.f;
+    return 5.f;
```

Comment:
Note we were already accounting for this value in the sizing methods, we just weren't using them in the layout (and until now it didn't matter because they were zero).